src/utils_request.js

```diff
@@ -180,14 +180,38 @@ export const requestAsBrowser = async (options = {}) => {
         };
     }
 
-    if (!gotScrapingOptions.isStream) return gotScraping(gotScrapingOptions);
+    if (!gotScrapingOptions.isStream) {
+        try {
+            return await gotScraping(gotScrapingOptions);
+        } catch (err) {
```

Comment:
Should we do this for all errors? Isn't there some type of error we can limit this to?